From 9ba702ce6eddda46fd07b3c91673661fc2f953dc Mon Sep 17 00:00:00 2001
From: Frank Wang <frank.wang@rock-chips.com>
Date: Tue, 29 Mar 2016 11:17:16 +0800
Subject: [PATCH] phy: rockchip-usb: Fixed the port cannot be resumed after
 suspended.

selecting utmi interface signals from utmi interface of usb20
host0 controller to usb2phy, when phy is resumed.

Change-Id: I487e836b89177cd8bc2dc56400f4dc277c8d2bf0
Signed-off-by: Frank Wang <frank.wang@rock-chips.com>
---
 drivers/phy/phy-rockchip-usb.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)
 mode change 100755 => 100644 drivers/phy/phy-rockchip-usb.c

diff --git a/drivers/phy/phy-rockchip-usb.c b/drivers/phy/phy-rockchip-usb.c
old mode 100755
new mode 100644
index 55d3ea07110b..28119edc0599
--- a/drivers/phy/phy-rockchip-usb.c
+++ b/drivers/phy/phy-rockchip-usb.c
@@ -45,7 +45,7 @@
 
 #define UTMI_SEL_GRF_WR_ENA	(0x3 << 16)
 #define UTMI_SEL_GRF_SUSPEND	(0x1 << 0)
-#define UTMI_SEL_GRF_RESUME	(0x3 << 0)
+#define UTMI_SEL_GRF_RESUME	(0x2 << 0)
 
 struct rockchip_usb_phys {
 	int reg;
-- 
2.35.3

